core/src/main/java/org/apache/iceberg/CatalogUtil.java

```diff
@@ -443,36 +443,42 @@ public static void configureHadoopConf(Object maybeConfigurable, Object conf) {
   public static MetricsReporter loadMetricsReporter(Map<String, String> properties) {
     String impl = properties.get(CatalogProperties.METRICS_REPORTER_IMPL);
     if (impl == null) {
+      LOG.info(
+          "Custom metrics reporter is set, but the implementation is null. Defaulting to LoggingMetricsReporter");
```

Comment:
I don't think this log message is actually correct. A custom metrics reporter isn't set, because `CatalogProperties.METRICS_REPORTER_IMPL` evaluates to `null`. So I'd rather just remove this log msg

Reply:
removed and all other code suggestions applied